src/HealthChecks/HealthChecks/src/DependencyInjection/HealthCheckServiceCollectionExtensions.cs

```diff
@@ -12,6 +13,44 @@ namespace Microsoft.Extensions.DependencyInjection
     /// </summary>
     public static class HealthCheckServiceCollectionExtensions
     {
+        /// <summary>
+        /// Adds the <see cref="HealthCheckService"/> to the container, using the provided delegate to register
+        /// health checks.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> to add the <see cref="HealthCheckService"/> to.</param>
+        /// <param name="configureOptions">A delegate to configure the <see cref="HealthCheckServiceOptions"/>.</param>
+        /// <returns>An instance of <see cref="IHealthChecksBuilder"/> from which health checks can be registered.</returns>
+        public static IHealthChecksBuilder AddHealthChecks(this IServiceCollection services, Action<HealthCheckServiceOptions> configureOptions)
+        {
+            if (configureOptions == null)
+            {
+                throw new ArgumentNullException(nameof(configureOptions));
+            }
+
+            services.AddOptions<HealthCheckServiceOptions>().Configure(configureOptions);
```

Comment:
Isn't it sufficient to call `services.Configure(configureOptions)`?

Reply:
Could you add a unit test for this?

Reply:
I added a test.